ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_15/InterfaceSubscriptionsIT.scala

```diff
@@ -717,6 +721,36 @@ class InterfaceSubscriptionsIT extends LedgerTestSuite {
     )
   }
 
+  private def hideTraceIdFromStatusMessages(
```

Comment:
This looks like a big workaround, I am wondering if we could just reduce the surface of comparison to number of transactions and some of their common properties (not sure what are these) rather than exclusing things here and there. Wdyt?

Reply:
https://github.com/digital-asset/daml/blob/157060fc7c924f52b0fb1e0939c2d71b766e8148/ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_15/InterfaceSubscriptionsIT.scala#L482-L497

The test wants to check that the `InterfaceView`s that are contained in the `Event`s of the `Transaction`s are identical.

However, every view contains a `ViewStatus`:
https://github.com/digital-asset/daml/blob/bca7488cc72dce5b3f66d4e5497131fa9ff279a0/ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto#LL133C1-L133C1

which seems to be significant for the comparison.

In the sense of lightening the test I could remove the `updateTransaction` and isolate only the views of the created events. Not overwriting the trace-id seems unavoidable to me.

Reply:
Do you think that checking only the number of views would suffice?

Reply:
Counting is not enough, you may have the same number but completely different content or subtly different content.